brave/brave-core
# Add metric for Ads clear data action

Resolves https://github.com/brave/brave-browser/issues/40548



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

0. Start with fresh profile, ensure metric does not exist in local state.
1. Open the browser history clearing dialog, tap the "Clear Brave Ads Data" button.
2. Ensure the metric exists with value 1.
